resources/mediaplayer/src/test/java/org/mobicents/media/server/impl/resource/mediaplayer/audio/wav/WavTrackBenchmarkTest.java

```diff
@@ -0,0 +1,180 @@
+
+/**
+ *
+ * @author apollo
+ */
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.concurrent.TimeUnit;
+import org.apache.log4j.Logger;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.mobicents.media.server.spi.memory.Frame;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.TimeValue;
+
+@State(Scope.Thread)
+public class WavTrackBenchmarkTest {
+    
+    @Param({"128", "512", "1024"})
+    private int buffSizes;
+
+    private static final int NUM_BYTES = 128;
+    private static Logger logger = Logger.getLogger(WavTrackBenchmarkTest.class);
+    private static URL url = WavTrackBenchmarkTest.class.getClassLoader().getResource("welcome.wav");
+    private ReadableByteChannel rbc;
+    
+    public WavTrackBenchmarkTest() {
+        
+    }
+
+    @Ignore
+    public void launchBechmark() throws Exception {
+
+            Options opt = new OptionsBuilder()
+                    // Specify which benchmarks to run. 
+                    // You can be more specific if you'd like to run only one benchmark per test.
+                    .include(this.getClass().getName() + ".*")
+                    // Set the following options as needed
+                    .mode (Mode.AverageTime)
+                    .timeUnit(TimeUnit.MICROSECONDS)
+                    .warmupTime(TimeValue.seconds(1))
+                    .warmupIterations(5)//10
+                    .measurementTime(TimeValue.seconds(1))
+                    .measurementIterations(5)
+                    .threads(5)
+                    .forks(1)
+                    .build();
+
+            new Runner(opt).run();
+        }
+    
+    @Benchmark
+    public void benchNioWavTrack() throws Exception {      
+        WavTrackImpl wavTrackImpl = new WavTrackImpl(url);
+        boolean isEOMReached = false;
+        while (!isEOMReached) {
+            Frame process = wavTrackImpl.process(0);
+            isEOMReached = process.isEOM();
+        }
+        wavTrackImpl.close();
+        
+    }
+
+    @Benchmark
+    public void benchWavTrack() throws Exception {      
+        WavTrack wavTrack = new WavTrack(url);
+        boolean isEOMReached = false;
+        while (!isEOMReached) {
+            Frame process = wavTrack.process(0);
+            isEOMReached = process.isEOM();
+        }
+        wavTrack.close();
```

Comment:
If an exception can be thrown, the wrap wavTrack.close() in a try-finally block, to guarantee resources are properly closed.